packages/core-svelte/src/lib/components/svelte-utils/TextComponentHelper.ts

```diff
@@ -287,6 +287,28 @@ export class TextComponentHelper {
         return this._getText() === '' || !this._getText();
     }
 
+        /**
+     * Converts camelCase text to readable text by adding spaces before capital letters
+     * and capitalizing the first letter.
+     * Example: "sourceToTargetMappings" -> "Source To Target Mappings"
+     * @param text The camelCase text to convert
+     * @returns The converted readable text
+     */
+    camelCaseToReadable(text: string): string {
+        if (!text || text.length === 0) {
+            return text;
+        }
+        
+        // Remove < and > characters
+        const withoutBrackets = text.replace(/[<>]/g, '');
```

Comment:
If the brackets are removed in the Helper classes (as in this PR), do we still need to replace them here?